### ISSUE TITLE
MDS 2.0 - accessibility_options for vehicles should be optional not required

### ISSUE THREAD

**Issue by lastalfriday** [openmobilityfoundation/mobility-data-specification#847]
The title says it all :)

https://github.com/openmobilityfoundation/mobility-data-specification/blob/2.0.0-rc1/data-types.md#vehicles

![image](https://user-images.githubusercontent.com/40355961/234219519-eb8875ee-1f07-4aa7-bb0b-efd6d35247e3.png)

**Comment by schnuerle** [comment]
Probably "Required if Available" is the way to go.

**Comment by thekaveman** [comment]
@schnuerle how about for Trips? `accessibility_options` is also listed as required there, should it be moved to "Required if Available"? https://github.com/openmobilityfoundation/mobility-data-specification/blob/release-2.0.0/data-types.md#trips

**Comment by lastalfriday** [comment]
Ok, I never really understood the idea behind "Required if available", but I'm guessing it has to do with encouraging providers to supply it if they have it..

Anyway, in the database it won't make a difference for me, it will still need to be "nullable".

"Naming things" is the hardest part of IT, so I have full understanding for this, **but** when you name something `xxx_options` and it's not optional, well...

**Comment by thekaveman** [comment]
> "Naming things" is the hardest part of IT, so I have full understanding for this, but when you name something `xxx_options` and it's not optional, well...

Agreed on "naming things" being difficult!! Though "options" here is being used as a synonym for "choices" - unrelated to "optional" / "required".

**Comment by lastalfriday** [comment]
Ok, I get it. Confusing though :) 
..could you not use *"attributes"* => `accessibility_attributes` ?   ...since it's used other places..

I also found a couple of other properties that should not be required, but optional or "required if available".
It makes no sense for a required property to have the value `null`

![image](https://user-images.githubusercontent.com/40355961/234811799-c4d9922f-2512-4e93-bdb4-ed88a1ac2083.png)

**Comment by lastalfriday** [comment]
A (non-related) question regarding `trip_ids` defined for telemetry? In what scenario does a telemetry point belong to two trips?

**Comment by schnuerle** [comment]
`accessibility_options` is really meant to list the options available for accessibility. It doesn't have anything to do with if the field is optional! You can have accessibility options available or not, so listing them is optional.

But I do think `accessibility_attributes` may be a better term here. I can make this change throughout.

For the side question of telemetry on multiple trips, the scenario came up in the working groups where the same trip in a car could be used to transport goods and/or people at the same time, under different billing and different customers. So trips can overlap within a mode or across multiple modes (like maybe food delivery at the same time as passenger delivery).

**Comment by lastalfriday** [comment]
Thanks for changing to `accessibility_attributes`, that's much more clear

Ok, I guess that could be a scenario, even though it seems a bit narrow, at least in our case.

The "required" bit should be changed to match events though?

**Comment by schnuerle** [comment]
I've got accessibility_options changed to accessibility_attributes in all places in the spec with this [commit](https://github.com/openmobilityfoundation/mobility-data-specification/pull/837/commits/ae19ac84b36c446d20b83aef64ab120b964349f6). With review by @thekaveman we can close this issue.

@lastalfriday if you want to continue discussing the overlapping trips telemetry, which I think warrants discussion, please open a new issue and include your summary and screenshots!

**Comment by thekaveman** [comment]
In reviewing Policy, I noticed there's a reference to Accessibility Options in the `policy.rule`: https://github.com/openmobilityfoundation/mobility-data-specification/blob/release-2.0.0/policy/README.md#rules

In addition to changing the field name there, it might make sense to elevate this field out of Rule and up to the Policy document. Since the `mode_id` is also at the Policy document level, and `accessibility_attributes` depend on `mode_id`.

**Comment by lastalfriday** [comment]
@schnuerle 
Thank you for renaming to `accessibility_attributes`. I see the changes live now in the `release-2.0.0` branch. 

However! I almost forgot this as we started discuss other things, my main point here was that this field should not be "Required", but either "Optional" or "Required if Available" (as you yourself suggested early on in this thread)

**Comment by schnuerle** [comment]
Thanks for the catch this is cleaned up [here](https://github.com/openmobilityfoundation/mobility-data-specification/pull/837/commits/b8fc6380faebac1a9706f3fea421409ca7b704a0).
